### PR TITLE
Image and input

### PR DESCRIPTION
**Pull request by preethasri**

### CODE REVIEW

**Review by vijaytembugade — APPROVED** [review]
image
You can check on google how can we select one radio button at a time. 

Rest things looks perfect. Style other things like navbar and sidebar, which will make look of website better.

File: image component/img.css [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+#side-heading-description {
+    font-size: 1.1rem;
+    margin-bottom: 1.5rem;
+}
+
+
+/* sidebar */
+
+.library-sidebar {
```
Comment (msdeshmukh009) [inline image component/img.css:33]: Hey preetha. I can see that you are writing sidebar styles in multiple stylesheets so it's better to make a separate stylesheet for that and then use it. A component stylesheet should only contain styles related to that respective component.

File: image component/img.css [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+
+/* header nav */
+
+#main {
+    transition: margin-left .5s;
+    padding: 16px;
+}
+
+#image-heading,
```
Comment (msdeshmukh009) [inline image component/img.css:88]: I think it's better to avoid id selectors as it is not reusable. Try classes for styling.

**Review by msdeshmukh009 — CHANGES_REQUESTED** [review]

File: image component/img.css (hunk 1 above)
Comment (preethasri) [reply image component/img.css:33]: > Hey preetha. I can see that you are writing sidebar styles in multiple stylesheets so it's better to make a separate stylesheet for that and then use it. A component stylesheet should only contain styles related to that respective component.

hey mahesh i thought i can do all these things after finishing the components. anyways thank you for pointing it out.will work on that 👍

**Comment by preethasri** [comment]
> image You can check on google how can we select one radio button at a time.
> 
> Rest things looks perfect. Style other things like navbar and sidebar, which will make look of website better.

will work on this vijay

File: image component/img.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        <div class="image-code-snippet">
+            <embed src="https://carbon.now.sh/embed?bg=rgba%28189%2C16%2C224%2C1%29&t=vscode&wt=none&l=htmlmixed&width=608.25&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=15.5px&lh=177%25&si=false&es=2x&wm=false&code=%253Cimg%2520src%253D%2522natureImg.jpg%2522%2520alt%253D%2522%2522%2520id%253D%2522res-image%2522%253E"
+                style="width: 608px; height: 85px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin" type="">
+        </div>
+        <div id="side-heading">Round Image</div>
+        <div id="side-heading-description">The main CSS property responsible for the circular shape is the border-radius property. Setting the radius of the corners to 50% of the width/height results in a circle.</div>
+        <div class="images">
+
+            <img src="/images folder/natureImg.jpg" alt="" id="round-image">
+
```
Comment (MiheerTamkhane) [inline image component/img.html:61]: I would recommend ad hiphen or underscore in between spaces of folder name.

**Review by MiheerTamkhane — APPROVED** [review]
Looks good to me, also try to make switchable radio buttons. Nice work

**Review by kiranpratyush — APPROVED** [review]

**Pull request merged**